src/components/card-link/index.tsx

```diff
@@ -9,11 +9,12 @@ const CardLink = ({
   children,
   className,
   href,
+  'aria-label': ariaLabel,
 }: CardLinkProps): ReactElement => {
   const classes = classNames(s.root, className)
 
   return (
-    <MaybeInternalLink href={href}>
+    <MaybeInternalLink href={href} aria-label={ariaLabel}>
```

Comment:
`CardLink` now supports `aria-label`, passing it to `MaybeInternalLink`.

I opted for snake casing here to match use on `<a />` tags (and therefore in `MaybeInternalLink`). Loosely held opinion is that the minor annoyance of having to use an alias when de-structuring props within `CardLink` is worth the consistency.

Reply:
✅ makes sense - i've renamed to `ariaLabel` for `CardLink`.